todo_app/app.py

```diff
@@ -1,28 +1,61 @@
+from board_item import Board_item
 from flask import Flask, render_template, redirect, url_for, request
 
 from todo_app.flask_config import Config
 from todo_app.data import session_items as session
 
+import requests
+import os
+from dotenv import load_dotenv
+import json
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
+load_dotenv()
+host =  os.getenv('HOST')
+api_key = os.getenv('API_KEY')
+api_secret = os.getenv('API_TOKEN')
+board_id = os.getenv('BOARD_ID')
+todo_board_list_id = os.getenv('TODO_LIST_ID')
+done_list_id = os.getenv('DONE_LIST_ID')
 
-@app.route('/')
-def index():
-    items = session.get_items()
-    return render_template('index.html', items = items)
 
+@app.route('/')
+def index():        
+    cards = requests.get( host + '/1/lists/' + todo_board_list_id + '/cards?key=' + api_key + '&token=' + api_secret).json()
+
+    display_cards = []
+    id_counter = 0
+    for card in cards:
+        name = card['name']
+        id = card['id']
+        status = card['dueComplete']
+        if status != True:
```

Comment:
Here you set the status then override it, which is a little confusing, it would be nicer to use an alternative variable name for the intermediate step, e.g.

```python
is_complete = card['dueComplete']
if not is_complete:
  ....
else:
  ...
```